### DISCUSSION TITLE
WIP: using lunamark with pandoc

### DISCUSSION THREAD

**Discussion by tarleb** [jgm/pandoc#9894] [Show and tell]
Lunamark is a Lua library and program for document conversions, written by jgm. The reason why I want to use it with pandoc is that lunamark comes with a nice, fast, and flexible Markdown parser.

The [pandoc-lunamark](https://github.com/tarleb/pandoc-lunamark) project provides a link between the two projects. It allows to use pandoc with the lunamark parser as a custom reader. The goal is to make it easy to implement custom Markdown syntax, using the lunamark parser as a solid foundation.

The project is very much work in progress and not production ready, but I'd love to get some early feedback.

Thanks!

**Comment by TomBener** [comment]
Great project. It would be great if there are specific examples provided to show how to use it.

**Reply by tarleb** [reply]
Thank you @TomBener! I will create a full example and then post an update here when it's ready.
